scripts/generate-relay-specs.sh

```diff
@@ -9,31 +9,16 @@ fi
 
 echo "Using Polkadot revision #${POLKADOT_VERSION}"
 
-docker run -it purestake/moonbase-relay-testnet:$POLKADOT_VERSION \
-  /usr/local/bin/polkadot \
-    build-spec \
-    -lerror \
-    --disable-default-bootnode \
-    --chain rococo-local \
-    | grep '\"code\"' > $POLKADOT_SPEC_TMP
-echo $POLKADOT_SPEC_TMP generated
-
-echo "Using $POLKADOT_SPEC_TEMPLATE..."
-sed -e "/\"<runtime_code>\"/{r $POLKADOT_SPEC_TMP" -e 'd;}' $POLKADOT_SPEC_TEMPLATE \
-  > $POLKADOT_SPEC_PLAIN
-echo $POLKADOT_SPEC_PLAIN generated
-
-
 # "Chain does not have enough staking candidates to operate" is displayed when no
 # staker is given at genesis
 
 docker run -it -v $(pwd)/build:/build purestake/moonbase-relay-testnet:$POLKADOT_VERSION \
   /usr/local/bin/polkadot \
     build-spec \
+    --chain rococo-local \
```

Comment:
How are we telling the relay to use our specs? I'm getting confused here :/

Reply:
After this PR there are no more plain chain spec json files commited to this repo. Anyone who wants a plain spec should use the node to generate it. This eliminates the possibility that the plain specs get out of sync with the code.

It seemed like all this script did was take a plain spec file that was previously committed and replace its wasm code with one from the new node. Why not just generate a new spec from the new node.

I considered removing this script entirely, but I figured it might break things I wasn't aware of so I left it.

Reply:
Oh you mean to generate the relay spec from the moonbeam node? I didn't know you could do that. 
I'm fine with that, as long as we have our custom specs (for dev and also for the (soon to be included) test relying on the relay).

Reply:
For posterity:

Alan and I agreed in slack that it makes sense to check in relay chain specs for integration tests when we have use for custom relay specs. For now we are fine using Alice and Bob, but if or when that changes, the necessary relay specs can be checked in here.

Thinking more about it, we should see whether we can get the collator node to generate relay specs for us. Maybe it wouldn't be so hard.